xarray/conventions.py

```diff
@@ -637,6 +665,13 @@ def maybe_encode_dtype(var, name=None):
                                   'any _FillValue to use for NaNs' % name,
                                   RuntimeWarning, stacklevel=3)
                 data = duck_array_ops.around(data)[...]
+                if '_Unsigned' in encoding:
```

Comment:
is `_Unsigned = False` a valid attribute?

If so, we need this to be `if encoding.get('_Unsigned', False)`.